### ISSUE TITLE
Related work

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/SSSOM#26]
This issue is to just dump some vaguely related work in; 

- [ ] http://grasstunes.net/ontology/similarity/0.2/musim.html
- [ ] http://alignapi.gforge.inria.fr/format.html
- [ ] http://alignapi.gforge.inria.fr/edoal.html
- [ ] https://www.fda.gov/drugs/coronavirus-covid-19-drugs/covid-19-real-world-data-rwd-data-elements-harmonization-project
- [ ] Provenance ontology
- [ ] [Similarity ontology](https://link.springer.com/content/pdf/10.1007%2F978-3-642-17746-0_20.pdf)
- [ ] [Biohackathon 2015](https://f1000research.com/articles/9-136) ([issue](https://github.com/OBOFoundry/SSSOM/issues/25)

**Comment by Chris-Evelo** [comment]
The BridgeDb project does related work on database ID mappings and mapping services. There too we need mappings and there provenance and mapping tools.

**Comment by cmungall** [comment]
Thanks @Chris-Evelo - do you have a link to the schema used for mappings in brridgedb, or the serialization format? cc @realmarcin

**Comment by matentzn** [comment]
Another important source, shared by @AlasdairGray: http://www.openphacts.org/specs/2013/WD-datadesc-20130912/

@cmungall you can probably derive the bridge-db schema from [Section A:1](http://www.openphacts.org/specs/2013/WD-datadesc-20130912/#bridgedb)..

**Comment by jhpoelen** [comment]
re: email thread related to contextual mappings with @matentzn @cmungall - Preston https://github.com/bio-guoda/preston, a biodiversity dataset tracker, (disclaimer: I am a contributor) uses Prov-O https://www.w3.org/TR/prov-o/ to describe the context in which datasets are tracked. See also https://doi.org/10.1016/j.ecoinf.2020.101132 . Preston maps URLs to unique content id/hashes to track published dataset versions.

**Comment by matentzn** [comment]
Thanks @jhpoelen, this also relates to https://github.com/OBOFoundry/SSSOM/issues/3

We will make alignment with PROV-O a priority once we have the basic syntactic/scope questions finalised! Thanks for sharing the references!

**Comment by Chris-Evelo** [comment]
Yes, @cmungall and @matentzn I think the Open PHACTS IMS version of BridgeDb indeed had most work done on the inclusion of provenance information (in the VoID header) and it allowed changes of predicates in the linksets. However in the scientific lenses approach (see this [pdf](https://link.springer.com/content/pdf/10.1007/978-3-319-11964-9_7.pdf) and these [slides](https://www.slideshare.net/alasdair_gray/scientific-lenses-over-linked-data-an-approach-to-support-multiple-integrated-views) ) approach ( we simply exchanged complete linksets, so in that for that purpose having the provenance and the type of mapping relationship on a set level is not even bad. For the traditional version of BridgeDb and the API we use Derby databases. But I think it would be nice to have a transport format that could be based on the IMS linksets and what we discuss here.

**Comment by matentzn** [comment]
https://www.omg.org/spec/DOL/1.0/PDF

Just mentioned by someone in @cmungall key note.

**Comment by matentzn** [comment]
SKOS play mentioned by @graybeal

**Comment by matentzn** [comment]
> Look at HL7 implementation profiles as a way to incorporate an approach to this complex mapping challenge

**Comment by matentzn** [comment]
Look at RELMA and how loinc does their mappings: https://loinc.org/mappings/

**Comment by matentzn** [comment]
Check out, also http://build.fhir.org/conceptmap2.html

**Comment by matentzn** [comment]
Check also SILK and its "Link Specification Language": https://app.assembla.com/wiki/show/silk/Link_Specification_Language

**Comment by matentzn** [comment]
see also #250

**Comment by matentzn** [comment]
Another one from @nichtich, see https://github.com/mapping-commons/sssom/discussions/197#discussioncomment-4694842: https://reconciliation-api.github.io/specs/latest/

**Comment by nichtich** [comment]
Are there any other tabular formats with header section? I know Markdown with YAML header (using `---` to separate YAML header from content) and the [DC Tabular Application Profile group](https://dcmi.github.io/dctap/) discussed use of header in tabular data but decided to put additional information [such as namespace prefixes](https://dcmi.github.io/dctap/TAPprimer.html#namespace-declarations) in separate documents (DCTAP has some more similarities to SSSOM Tables but different application). I've also seen CSV with multi-line header rows but the number of header lines is known in advance. Do other tabular formats use `#` as header indicator?

**Comment by graybeal** [comment]
Oh yeah! The Excel-to-SKOS converter format in [SKOSPlay](https://skos-play.sparna.fr/play/) is outstanding, and also can convert Excel to OWL if you do things right. A colleague and I have created a whole automated GitHub-based pipeline [excel2rdf](https://excel2rdf.readthedocs.io), and a companion one based on Google docs (sheet2rdf). I can not promote the quality of the SKOS Play software highly enough, I consider it a beautiful piece of work. 

Here is an [example Google sheet](https://docs.google.com/spreadsheets/d/19cnChnzukBQv5uvPxmyWB9am_YvnONVrvjm87dhpT34/edit#gid=1198865354) (slightly overcolored!) to illustrate a detailed self-documenting example. 

(I've tried to convince SSSOM to make their format SKOSPlay-compatible, it's 95% there and it would make transformation to RDF trivial. But for some reason no one besides me gets excited about that. :-(  )

**Comment by matentzn** [comment]
I think it is because no one in our world uses SKOS Play :) Building a transformer should be easy enough!

**Comment by graybeal** [comment]
Spreadsheet is now viewable, sorry!

**Comment by matentzn** [comment]
I can see the appeal for such a format for human reviewers/curators, but its not very easily machine processable by standard data science tools. However, it should not be hard to write a converter. In this case, I do prefer the current design in SSSOM, despite the obvious advantages for human readbility of the table you shared!

**Comment by Chris-Evelo** [comment]
So maybe a converter in the other direction would be a good thing? In that way, you would always have the SSSOM design which is nicely machine-readable, but for inspection, you could create a more human-friendly format.

**Comment by matentzn** [comment]
This is an open ended problem out of scope for SSSOM, but continued by the [RDA FAIR Mappings WG](https://github.com/mapping-commons/rda-fair-mappings).

**Comment by matentzn** [comment]
Moved here: https://github.com/mapping-commons/rda-fair-mappings/issues/49
